SU-HSDO/suhumsci
# Release: Four Kitchens Sprint 7

## Summary
- feat: firefinch color pairing
- feat: align "more" links that can be attached to views to the right
- fix: mailto links not always applying the correct link color
- fix: consistent image/hero height across all slides on mobile
- fix: consistent controls placement for hero sliders within the same row

## Need Review By (Date)
- By next release

## Urgency
- low

## Steps to Functional Test
1. Review email links that appear in the footer of most sites (music specifically) and verify they have a consistent link color.
2. Go to https://hs-colorful.suhumsci.loc/ (homepage of hs-colorful site) and verify the "more link" for the Spotlight news section is aligned to the right of its container.
3. Go to the theme settings page admin/appearance/settings/humsci_colorful, check 'Use Animation Enhancements'. Go to /components/hero-images-main-region and verify that the text displays for the Hero (Gradient) (first) and the Hero (Overlay) (third) component
4. For the same page as above, add two or three slides with different image aspect ratios and different text lengths to a `Hero (Layered) - Slider`. Scroll through the slider at mobile to observe that: the image takes the same height and the box container for the title/text takes the same height across slides.
5. Go to https://hs-traditional.suhumsci.loc/ (homepage of traditional site). Go to the settings for Traditional and select 'Firefinch' as the 'Color Pairing'. Verify the new color pairing applies to the frontend of the traditional theme.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)

Merged PRs:
- https://github.com/SU-HSDO/suhumsci/pull/1154
- https://github.com/SU-HSDO/suhumsci/pull/1155
- https://github.com/SU-HSDO/suhumsci/pull/1156
- https://github.com/SU-HSDO/suhumsci/pull/1157
- https://github.com/SU-HSDO/suhumsci/pull/1159
- https://github.com/SU-HSDO/suhumsci/pull/1160
